tests/test_misc.py

```diff
@@ -178,21 +178,75 @@ def test_sigfetcher(revault_network, bitcoind, executor):
     for w in rn.stks():
         w.wait_for_logs(
             [
-                "Fetching Unvault Emergency signature",
-                "Fetching Emergency signature",
-                "Fetching Cancel signature",
-                "Fetching Unvault signature",
+                "Syncing Unvault Emergency signature",
+                "Syncing Emergency signature",
+                "Syncing Cancel signature",
+                "Syncing Unvault signature",
             ]
         )
     for w in rn.man_wallets:
         w.wait_for_logs(
             [
-                "Fetching Cancel signature",
-                "Fetching Unvault signature",
+                "Syncing Cancel signature",
+                "Syncing Unvault signature",
             ]
         )
 
 
+@pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
+def test_sigfetcher_coordinator_dead(revault_network, bitcoind):
+    rn = revault_network
+    rn.deploy(
+        2,
+        1,
+        csv=1,
+    )
+    vault = revault_network.fund(1)
+
+    # We kill the coordinator (and cosigners/watchtowers as well, but who cares?)
```

Comment:
`with_cosigs=False` and `with_wts=False` would probably speed this up :)

Reply:
Also if it was configured with watchtowers a revaultd should error to share revocation txs on this basis rather than the coord being dead?